common/religion/religions/water_group.txt

```diff
@@ -278,16 +278,35 @@
 			holy_site = yulon
 			holy_site = xuen
 			
+			doctrine = special_doctrine_celestial_communion
+			
 			#Main Group
 			doctrine = doctrine_no_head
+			doctrine = doctrine_pluralism_righteous
```

Comment:
Maybe make them and Celestial pluralistic?

Reply:
I can see it either way but am leaning towards righteous for both. Pandaren don't strike me as a particularly bigoted people but they are very insular and arguably conservative in traditions, this is already reflected in the magic doctrines where everything except elemental practice is ignored or illegal. The Jinyu are more firmly in the righteous camp given their structured caste based society which is practically built around the cult of waterspeakers.